src/auto-updater/index.js

```diff
@@ -7,8 +7,14 @@ const { showDialog } = require('../dialogs')
 const { IS_MAC, IS_WIN, IS_APPIMAGE } = require('../common/consts')
 const ipcMainEvents = require('../common/ipc-main-events')
 const getCtx = require('../context')
+const store = require('../common/store')
+const CONFIG_KEYS = require('../common/config-keys')
 
 function isAutoUpdateSupported () {
+  if (!store.get(CONFIG_KEYS.AUTO_UPDATE, true)) {
```

Comment:
could we flip this and have `DISABLE_AUTO_UPDATE` instead? (we don't want to disable autoupdates due to a bug where there is a typo in key  or the key is missing (`!undefined` → `true` :grimacing: )

```suggestion
  if (store.get(CONFIG_KEYS.DISABLE_AUTO_UPDATE, false)) {
```